lib/toString.js

```diff
@@ -43,7 +45,7 @@ function toString () {
   plugins.forEach((plugin, i) => {
     outputString += `\n${
       i < plugins.length - 1 ? LINES.t : LINES.r
-    } ${plugin.name.length === 0 ? 'anonymous' + i : plugin.name}`
+    } ${plugin.name.length === 0 ? plugin[Symbol.for('fastify.display-name')] : plugin.name}`
```

Comment:
I think we should also check if neither of these are available and just use `'anonymous'` as a fallback.